src/instructlab/cli/model/evaluate.py

```diff
@@ -135,6 +135,31 @@
     is_flag=True,
     help="Print serving engine logs.",
 )
+@click.option(
+    "--input-questions",
+    type=click.Path(exists=True, path_type=pathlib.Path),
+    default=None,
+    help="Path to the questions and reference answers the model will be evaluating in the DK-Bench evaluation",
+)
+@click.option(
+    "--output-file-formats",
+    type=click.STRING,
+    default="jsonl",
+    show_default=True,
+    help="Comma-separated list of file formats for results of the DK-Bench evaluation. Valid options are csv, jsonl, and xlsx. If this option is not provided the results are written as a .jsonl file",
+)
+@click.option(
+    "--system-prompt",
```

Comment:
these flags with codified defaults should be reading from `configuration.py`. There should either be a new class, or additions to the eval class with these defaults in every config.